libp2p/services/hpservice.nim

```diff
@@ -32,10 +32,10 @@ type
     autonatService: AutonatService
     isPublicIPAddrProc: IsPublicIPAddrProc
 
-  IsPublicIPAddrProc* = proc(ta: TransportAddress): bool {.gcsafe, raises: [].}
+  IsPublicIPAddrProc* = proc(ma: MultiAddress): bool {.gcsafe, raises: [].}
```

Comment:
Is it still necessary btw? Seems we can now use `/dns/localhost` in test to emulate a public node

Reply:
True, but it's a bit cumbersome to set it.

Reply:
Done.